### ISSUE TITLE
Express dependency needs upgrade in @nestjs/platform-express

### ISSUE THREAD

**Issue by mo-alaa** [nestjs/nest#14061]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

The `express` dependency depends on the `cookie` package which has a security vulnerability: [CVE-2024-47764](https://www.mend.io/vulnerability-database/CVE-2024-47764)
This is solved in cookie version 0.7.0, i.e. express upgrade to the latest version: `4.21.1` which is why this dependency needs updating.

### Minimum reproduction code

https://www.mend.io/vulnerability-database/CVE-2024-47764

### Expected behavior

According to your policy security vulnerabilities should be sent via email, but I sent an email but got no response. That's why I opened this ticket. 

Thank you :) 

### Package

- [ ] I don't know. Or some 3rd-party package
- [ ] <code>@nestjs/common</code>
- [ ] <code>@nestjs/core</code>
- [ ] <code>@nestjs/microservices</code>
- [X] <code>@nestjs/platform-express</code>
- [ ] <code>@nestjs/platform-fastify</code>
- [ ] <code>@nestjs/platform-socket.io</code>
- [ ] <code>@nestjs/platform-ws</code>
- [ ] <code>@nestjs/testing</code>
- [ ] <code>@nestjs/websockets</code>
- [ ] Other (see below)

### NestJS version

10.4.4

### Packages versions

```json
"@nestjs/common": "^10.4.4",
"@nestjs/core": "^10.4.4",
"@nestjs/platform-express": "^10.4.4",
```

### In which operating systems have you tested?

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by ezintz** [comment]
Hello, I already created a pull request for those: https://github.com/nestjs/nest/pull/14060

**Comment by wahidfebr** [comment]
@nestjs/platform-fastify also have this issue

**Comment by kamilmysliwiec** [comment]
Fixed in v10.4.5

**Comment by ezintz** [comment]
@kamilmysliwiec I just checked it but unfortunately it seems that it platform-express is still delivering 4.21.0

```shell
mkdir tmp
npm init -y
npm i --save @nestjs/platform-express
cat node_modules/express/package.json
```

to me it still shows 4.21.0 :( can you please have a look again? that would be awesome, thank you!
